spec/requests/api/client_can_send_payment_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe 'POST /api/subscriptions', type: :request do
+  describe 'successfully' do
```

Comment:
What happens if its not successful?

Reply:
I'll try to write more sad path for testing. Thanks